drivers/wifi/nrf700x/osal/fw_if/umac_if/src/tx.c

```diff
@@ -810,48 +828,32 @@ unsigned int tx_buff_req_free(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 			      unsigned int tx_desc_num,
 			      unsigned char *ac)
 {
-	struct wifi_nrf_fmac_priv *fpriv = NULL;
 	unsigned int pkts_pend = 0;
 	unsigned int desc = tx_desc_num;
 	int tx_done_q = 0, start_ac, end_ac, cnt = 0;
-	unsigned int queue_map = fmac_dev_ctx->tx_config.spare_desc_queue_map;
-
-	fpriv = fmac_dev_ctx->fpriv;
+	unsigned short tx_done_spare_desc_q_map = 0;
 
 	/* Determine the Queue from the descriptor */
 	/* Reserved desc */
-	if (desc < (fpriv->num_tx_tokens_per_ac * WIFI_NRF_FMAC_AC_MAX)) {
+	if (desc < (fmac_dev_ctx->fpriv->num_tx_tokens_per_ac * WIFI_NRF_FMAC_AC_MAX)) {
 		/* Derive the queue here as it is not given by UMAC.
-		 * tx_done_q = desc
+		 pare_desc_q_map & (1 << WIFI_NRF_FMAC_AC_BK)) tx_done_q = desc
```

Comment:
Is this a inadvertent paste here?

Reply:
Mistakenly it was added. Replaced to correct place.